src/Products/PloneMeeting/migrations/migrate_to_4217_1.py

```
@@ -33,21 +33,41 @@ def _configureCssTransforms(self):
             delattr(cfg, 'hideCssClassesTo')
         logger.info('Done.')
 
+    def _updateGroupsInChargeNotes(self):
+        """Update config and init new fields related to groupsInChargeNotes."""
+        logger.info('Updating "itemFieldsConfig" about groupsInChargeNotes for every MeetingConfigs...')
+        # update MeetingConfig.itemFieldsConfig default for "groupsInChargeNotes"
+        for cfg in self.tool.objectValues('MeetingConfig'):
+            item_fields_config = cfg.getItemFieldsConfig()
+            if not "groupsInChargeNotes" in [row['name'] for row in item_fields_config]:
```

Ruff (0.15.11)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)